src/global.json

```diff
@@ -1,4 +1,8 @@
 {
+    "sdk": {
+        "version": "7.0.400",
+        "rollForward": "latestMinor"
```

Comment:
Do we need this, suspect you might be doing it due to having .net 8 installed locally?

Just locks us into one version.

Reply:
We need it for the docs, if you don't specify a minor version it actually ignores the value, if there's a 7.1 released then we can come back and update as required, we have roll forward on the minor so will always be the latest version of sdk 7.0.xxx, this uses the same pwsh sdk installer as the github action sdk installer and requires 3 parts to the property V.Ma.Mi